platform/influxdb.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         ports:
         - containerPort: 8086
         env:
-        - name: INFLUXDB_LOGGING_LEVEL
+        - name: INFLUXDB_LOG_LEVEL
```

Comment:
InfluxDB uses "logging" in its variable declarations

https://docs.influxdata.com/influxdb/v1.8/administration/config/#logging-settings